### ISSUE TITLE
Improve dimensionality reduction example

### ISSUE THREAD

**Issue by rodrigobarongavb** [GAVB-SERVICOS/Gumly#12]
Use [make_classification](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_classification.html) to create an dataset and apply the dimensionality reduction methods, compare them and plot the comparison of number of components and their accuracy!
